libs/designsystem/button/src/button.component.scss

```diff
@@ -10,14 +10,24 @@ $button-width: (
   md: 88px,
   lg: 220px,
 ) !default;
+$button-margin: utils.size('xxxs');
 
 @function button-width($key) {
   @return map.get($button-width, $key);
 }
 
 @mixin button-xs {
+  /*
+  * Extra small button needs a smaller click area to e.g. not overlap with other 
+  * interactive elements. The maximum target size we can safely do is the button's 
+  * minimum width plus its margin on each side
+  */
+  $button-xs-icon-only-size: utils.size('m');
+  $button-xs-click-area-size: $button-xs-icon-only-size + $button-margin * 2;
+  @include utils.accessible-target-size($width: $button-xs-click-area-size);
+
   font-size: utils.font-size('xs');
-  height: utils.size('m');
+  height: $button-xs-icon-only-size;
```

Comment:
Is this also a problem vertically? 🤔

Reply:
Yes, we want the click area to be as big as possible both vertically and horizontally to get as close to 44 x 44 pixels as we can. 

In this case though, I am just extracting the value to a variable so I can reuse it for both height, width and the calculations.